### DISCUSSION TITLE
I am not able to use updated lifecylcle hook of livewire

### DISCUSSION THREAD

**Discussion by dhruva81** [rappasoft/laravel-livewire-tables#706] [Q&A]
Hi 

How can I use updated lifecycle hook of livewire on filters.

I am trying to get dynamic select filter values based on values selected in first filter. (Dependent chained filters). 
In v1 it was working fine but in v2 i am getting following exception.

```
Declaration of App\Http\Livewire\UsersDatatable::updated() must be compatible with Rappasoft\LaravelLivewireTables\DataTableComponent::updated($name, $value): void

```

Please help.

**Comment by rappasoft** [comment]
I'm using that method. Try constructing the parent method to see if it works.

**Reply by dhruva81** [reply]
Thank you for reply.

Sorry... I am not able to figure out how to construct the parent method.
Kindly guide me. If possible kindly share some snippet for this.

Thank you

**Reply by rappasoft** [reply]
This is untested but:

```
public function updated($name, $value): void
{
    parent::__construct();

    // Your code
}
```

**Reply by dhruva81** [reply]
Thank you a lot. 
It is working fine 👍 💯 

But I am facing another issue...
When I select any filter, my filter popover menu closes and I have to again click on filters button to open the popover menu .
This behaviour is same for slide-down filters.
